packages/core/src/components/MentionSender/index.vue

```
@@ -95,22 +97,52 @@ function onContentMouseDown(e: MouseEvent) {
+  }
+});
+
+/**
+ * 打开头部容器
+ * @deprecated 此方法将在下个大版本中移除，请使用 v-model:openHeader 代替
+ * @returns {boolean} 是否成功打开
+ */
 function openHeader() {
-  if (!slots.header)
-    return false;
+  if (!slots.header) return false;
```

ESLint

[error] Expect newline after if

(antfu/if-newline)